### DISCUSSION TITLE
iOS 17-18 support

### DISCUSSION THREAD

**Discussion by shivank-gupta-01** [YueChen-C/py-ios-device#162] [Q&A] [answered]
Need to know does `py-ios-device` is supporting newer iOS versions v17 and v18?

**Comment by YueChen-C** [comment]
https://github.com/YueChen-C/py-ios-device?tab=readme-ov-file#support-ios-17-command-line-is-not-supported-

**Reply by shivank-gupta-01** [reply]
On the basis of above, can you please clear if iOS 18 supported by `py-ios-device`?

**Reply by YueChen-C** [reply, marked as answer]
yes All support
